freesitemgr

```diff
@@ -271,6 +271,10 @@ def help():
     print("          - index file (default is index.html)")
     print("  -m, --mime-type")
     print("          - mime-type of the index file (default is \"text/html\")")
+    print("  --mime-type-match=PATTERN=MIME/TYPE")
+    print("     Set mime-type for files matching PATTERN. Can be given")
+    print("     multiple times to match distinct patterns.")
```

Comment:
There are a lot more to say about this but I don't think this is the place.

Among other things:
- If no files match, the mime-type will be determined my fcp3/node.py guestMimeType(filename) that in turn uses mimetypes.guess_type(filename, False). 
- The order is significant, i.e. --mime-type-match=*.gif=image/gif --mime-type-match=databasefiles/*=text/plain will, for the file databasefiles/somedir/*.gif give the mime-type image/gif while --mime-type-match=databasefiles/*=text/plain --mime-type-match=*.gif=image/gif will give the mime-type text/plain for the same file name.

Reply:
Can you add that information in a file in doc/? Maybe with an example how to use this to upload a repo for dgof? Maybe as doc/freesitemgr.org ? (in org-mode format — markdown would also work)

Reply:
OK, yes. I will see if I understand the org-mode format. I will do that in another pull request.